### PR TITLE
README Korean translation added

### PR DESCRIPTION
**Pull request by OctopusET**
Hello, I translated README in korean.

I'm working on translating index page now.

Thank you.

### CODE REVIEW

File: README_KO.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+우리의 목소리를 굳건하면서도 전문적으로 유지합시다.
+
+가능하다면 본 공개서한을 포럼이나 SNS에 공유해 주시고, 이 문제에 대해 도움이 될만한 기자들에게 알려주십시오.
+
+**Pull requests 는 12시간 내에 merge 됩니다 - 많은 양의 PR에 따라, PR들은 일괄적으로 처리됩니다**
+
+## Chatrooms
+
+- **Matrix.org:** #free-rms:matrix.org
+- **IRC:** #free-rms at chat.freenode.net
```
Comment (imachug) [inline README_KO.md:33]: Can you please translate the `at` preposition too?

Comment (OctopusET) [reply README_KO.md:33]: Okay

Comment (OctopusET) [reply README_KO.md]: Done

**Comment by imachug** [comment]
Thanks a lot!

**Pull request merged**